gym/core.py

```diff
@@ -1,3 +1,5 @@
+import numpy as np
```

Comment:
do we need to import it here? looks like Space.__init__ intentionally uses lazy load of numpy